tutorials/configure-virtual-machine-esxi/index.mdx

```diff
@@ -53,7 +53,7 @@ EXSi can be installed on a Scaleway Dedibox server in two ways: through the [Ded
 2. Click **VIRTUALIZATION DISTRIBUTIONS**:
   <Lightbox src="scaleway-virtualizaton-distributions.png" width="800" alt="" />
 
-3. Select EXSi on the distribution menu, then **EXSi 6.5.0d 64BITS** under Chose a Version and click **Install EXSI**:
+3. Select EXSi on the distribution menu, then **EXSi 6.5.0d 64BITS** under **Choose a Version** and click **Install EXSI**:
```

Comment:
Is 6.5.0d still the latest version available ?

Reply:
Fixed, tested and updated for 7.0